profiles/targets/clang/make.defaults

```diff
@@ -0,0 +1,6 @@
+# Copyright 1999-2016 Gentoo Foundation.
+# Distributed under the terms of the GNU General Public License, v2
+# $Id$
+
+CC="clang"
+CXX="clang++"
```

Comment:
Confirmed. It's just that I don't see the BUILD_CC/BUILD_CXX stuff documented anywhere...